content/sensu-core/2.0/getting-started/installation-and-configuration.md

```diff
@@ -6,196 +6,286 @@ weight: 1
+-p 3000:3000 -p 8080:8080 -p 8081:8081 sensu/sensu:2.0.0-beta.3 sensu-backend start
 {{< /highlight >}}
 
-Install the packages from the Sensu beta repository.
-
+2. Start an agent.
+In this case, we're starting an agent whose ID is the hostname with the webserver and system subscriptions. This assumes that sensu-backend is running on another host named sensu.yourdomain.com. If you are running these locally on the same system, be sure to add `--link sensu-backend` to your Docker arguments and change the backend URL `--backend-url ws://sensu-backend:8081`.
 {{< highlight shell >}}
-sudo apt-get install sensu-backend sensu-agent
+docker run -v /var/lib/sensu:/var/lib/sensu -d --name sensu-agent \
+sensu/sensu:2.0.0-beta.3 sensu-agent start --backend-url ws://sensu.yourdomain.com:8081 \
+--subscriptions webserver,system --cache-dir /var/lib/sensu
 {{< /highlight >}}
 
-{{< platformBlockClose >}}
+_NOTE: You can configure the sensu-backend and sensu-agent log levels by using the `--log-level` flag on either process. Log levels include `panic`, `fatal`, `error`, `warn`, `info`, and `debug`, defaulting to `warn`._
+
+### sensuctl and Docker
 
+It's best to run sensuctl locally and point it at the exposed API port for your sensu-backend process. The sensuctl utility stores configuration locally, and you'll likely want to persist it across uses. While it can be run from the docker container, doing so may be problematic.
+{{< platformBlockClose >}}
 {{< platformBlock "RHEL/CentOS" >}}
-### RHEL/CentOS
+## RHEL/CentOS
+
+- [Install the Sensu backend](#rhel-backend)
+- [Install the Sensu command line interface (sensuctl)](#rhel-cli)
+- [Install the Sensu agent](#rhel-agent)
 
-Add the Sensu beta repository.
+### Install the Sensu backend {#rhel-backend}
 
+1. Add the Sensu beta repository.
 {{< highlight shell >}}
 curl -s https://packagecloud.io/install/repositories/sensu/beta/script.rpm.sh | sudo bash
 {{< /highlight >}}
 
-Install the Sensu backend and agent packages.
-
+2. Install the Sensu backend.
 {{< highlight shell >}}
-sudo yum install sensu-backend sensu-agent
+sudo yum install sensu-backend
 {{< /highlight >}}
-{{< platformBlockClose >}}
-
-{{< platformBlock "Windows" >}}
-### Windows
-
-Download the [Sensu agent binary](https://storage.googleapis.com/sensu-binaries/2.0.0-beta.4-1/windows/amd64/sensu-agent).
-{{< platformBlockClose >}}
 
-## Configuration
+3. Copy the example backend configuration file to the default configuration path.
+{{< highlight shell >}}
+sudo cp /etc/sensu/backend.yml.example /etc/sensu/backend.yml
+{{< /highlight >}}
 
-The example config files list all of the configurable options for each service.
+4. Start the backend.
+{{< highlight shell >}}
+sudo systemctl start sensu-backend
+{{< /highlight >}}
+_NOTE: For CentOS 6 and RHEL 6, use `sudo /etc/init.d/sensu-backend start`._
+Access the Sensu dashboard at TCP port 3000, and log in as the default administrative user
```

Comment:
Change to "Access the Sensu dashboard at port 3000 in the browser of your choice"